charts/kyverno-policies/values.yaml

```diff
@@ -0,0 +1,5 @@
+# Supported- baseline/restricted/privileged/custom
```

Comment:
This looks like boilerplate from the upstream kyverno policies repository. As we are not using the upstream, what is the value of this here?

Reply:
We don't have any categories as such. Can we just keep baseline or remove it as well?